### PR TITLE
feat: Add Swift Sub Category

### LINKED ISSUE #2148: [Add] <Official doc of Swift Programming Language>
### Site URL

https://www.swift.org/documentation/

### Description of the above provided site

The official documentation of Swift Programming Language provides comprehensive information, guides, and resources for developers using Swift for iOS, macOS, watchOS, and tvOS application development. It covers the language syntax, standard library, and various features of Swift.

### Category

Resources

### Sub Category

documents

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by ketanbaitule**
## Fixes Issue

Closes #2148 

## Changes proposed

- Add Sub Category in `data.ts` , `index.ts`
- Create swift.json
- Added Sub Category Description For Swift in CategoryDescriptions.ts

## Screenshots

image

image

## Note to reviewers

Suggest any required changes!!
Thank You.

### CODE REVIEW

File: database/languages/swift.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "name": "Swift Official Documentation",
+        "description": "The official documentation of Swift Programming Language provides comprehensive information, guides, and resources for developers using Swift for iOS, macOS, watchOS, and tvOS application development. It covers the language syntax, standard library, and various features of Swift.",
```
Comment (CBID2) [inline database/languages/swift.json:4]:
```suggestion
        "description": "The official documentation of Swift. It provides comprehensive information, guides, and resources for developers using Swift for iOS, macOS, watchOS, and tvOS application development. It also covers the language syntax, standard library, and various features of Swift.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ketanbaitule. Your PR is of to be a pretty good start! :) It just needs some revision.

**Review by CBID2 — APPROVED** [review]
Looks good to me @ketanbaitule

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
Lgtm

**Pull request merged**